src/main/java/org/c02e/jpgpj/Encryptor.java

```diff
@@ -408,13 +411,68 @@ public void encrypt(InputStream plaintext, OutputStream ciphertext)
      * @throws PassphraseException if an incorrect passphrase was supplied
      * for one of the signing keys.
      */
-    public void encrypt(InputStream plaintext, OutputStream ciphertext,
-    FileMetadata meta) throws IOException, PGPException {
+    public void encrypt(
+        InputStream plaintext, OutputStream ciphertext, FileMetadata meta)
+            throws IOException, PGPException {
+        if (meta == null) {
+            meta = new FileMetadata();
+        }
+
+        try (OutputStream targetStream = prepareCiphertextOutputStream(ciphertext, meta)) {
+            // copy plaintext bytes into encryption pipeline
+            copy(plaintext, targetStream, null /* signer is inside the target */, meta);
+        }
+    }
+
+    public OutputStream prepareCiphertextOutputStream(File ciphertext)
```

Comment:
Do you think it's worthwhile having both this method and the prepareCiphertextOutputStream(OutputStream) method as part of the public API? If so, could you also add javadocs to this method, as well as a test for it in the EncryptorSpec (something like the "encrypt and sign file" test)?

Reply:
Will do...